### DISCUSSION TITLE
Why no recurrence?

### DISCUSSION THREAD

**Discussion by JeroenJochems** [spatie/laravel-google-calendar#284] [General]
Is there a good reason why recurrence was not implemented here?  Looking at the google calendar API it seems like adding recurrence to this package would be rather trivial. Am I missing something?

**Comment by EmilioPuljiz** [comment]
I am needing to implement recursion, is there a way? or it still cannot be done?

add these lines of code in Event.php

`public function setRecurrenceRule($recurrence)
    {
        $this->googleEvent->setRecurrence([$recurrence]);
        return $this;
    }`
    
When you are going to create the event, what I do is pass the following to it:

`$event->setRecurrenceRule('RRULE:FREQ=WEEKLY;COUNT=5');`
